### PR TITLE
Bug fixes, added documentation

### PR DESCRIPTION
**Pull request by superMDguy**
I fixed some bugs I found when trying to create an app, and added in the documentation for GFlags.

### CODE REVIEW

File: farg/core/ui/gui/central_pane.py [hunk 1]
```diff
@@ -23,96 +23,100 @@
 from tkinter import ALL, Canvas, Menu
 
 gflags.DEFINE_string('gui_initial_view', '',
-                     'In GUI mode, what should the initial mode be?')
+										 'In GUI mode, what should the initial mode be?')
```
Comment (amahabal) [inline farg/core/ui/gui/central_pane.py]: The indentation here seems strange.

File: farg/core/ui/gui/central_pane.py [hunk 2]
```diff
@@ -23,96 +23,100 @@
...
+	#: viewports, each an instance of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
+	#: Methods are available in this class to create views (e.g., SetFullView and
+	#: SetVerticallySplitView). See farg.apps.seqsee.gui.gui.py for an example.
+	named_views = {}  # Not a constant. pylint: disable=C6409
+
+	#: Name of initial view.
+	default_initial_view = ''  # Not a constant. pylint: disable=C6409
+
+	def __init__(self, master, controller, *, height, width, background):
+		self.height = height
```
Comment (amahabal) [inline farg/core/ui/gui/central_pane.py]: And the indentation in this class. Are you following a particular scheme? Can we stick to indenting two spaces (or 4, if you prefer that).

File: farg/core/ui/gui/central_pane.py [hunk 3]
```diff
@@ -23,96 +23,100 @@
...
+	def SetThreeWaySplit(self, view_class1, view_class2, view_class3):
+		"""Set central pane to contain a three view, one on top and two on the bottom.
+
+		Args:
+			view_class1: A subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
+			view_class2: Another subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
+			view_class3: Also a subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
+		"""
+		self.viewports = [view_class1(self, 0, 0, self.width, self.height / 2 - 2),
+											view_class2(self, 0, self.height / 2 + 2,
```
Comment (amahabal) [inline farg/core/ui/gui/central_pane.py]: Again indentation, and a blank line next.

File: farg/third_party/skeleton/__init__.py [hunk 4]
```diff
@@ -4,7 +4,9 @@
 
 """
 
+
 from farg.third_party.skeleton.core import ( 
+
```
Comment (amahabal) [inline farg/third_party/skeleton/__init__.py]: Is that empty line intentional?

**Comment by superMDguy** [comment]
Sorry about that, I'll fix it.

**Comment by superMDguy** [comment]
That should be ok.

**Pull request merged**